packages/shared/src/components/Icon.tsx

```diff
@@ -1,13 +1,14 @@
 import React, { ReactElement } from 'react';
 import classNames from 'classnames';
 
-type Size = 'small' | 'medium' | 'large' | 'xlarge';
+type Size = 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge';
 
 const IconSize: Record<Size, string> = {
   small: 'w-5 h-5',
   medium: 'w-6 h-6',
-  large: 'w-8 h-8',
-  xlarge: 'w-10 h-10',
+  large: 'w-auto w-auto',
```

Comment:
I'm not sure being large has auto values. It may happen where it can even be bigger than the `xlarge`. I think you should introduce one more size being named `auto`.

Reply:
It was a part of hacky way of adding large size (complete style was w-auto h-auto and fontSize: 1.75rem), but now we have w-7 h-7